cfgov/jinja2/v1/_includes/molecules/contact-phone.html

```diff
@@ -37,12 +37,15 @@
     <span class="h5">{{ label }}</span>
     {% for phone in value.phones %}
     <p>
-        <b>{{ format_phone(phone.number) }}</b>
+        <b>
+            {{ format_phone(phone.number) }}
+            {{ 'Ext ' ~ phone.extension if phone.extension else '' }}
+        </b>
         {% if phone.vanity %}
         <span>{{ format_phone(phone.vanity) }}</span>
         {% endif %}
         {% if phone.tty %}
-        <span>TTY/TDD: {{ format_phone(phone.tty) }}<span>
+        <span>TTY/TDD: {{ format_phone(phone.tty) }}{{ ' Ext ' ~ phone.tty_ext if phone.tty_ext else '' }}<span>
```

Comment:
No period at the end of `Ext` is stylistically correct @sarahsimpson09 ?

Reply:
No, you're right, it should be "Ext."